main.go

```diff
@@ -17,14 +17,17 @@ import (
 
 	"regexp"
 
+	"crypto/tls"
+	"net/http"
+
 	"golang.org/x/crypto/ssh/terminal"
 
 	log "github.com/sirupsen/logrus"
 	"github.com/docker/distribution/context"
 	schema2 "github.com/docker/distribution/manifest/schema2"
 	"github.com/docker/distribution/reference"
 	"github.com/docker/distribution/registry/client"
-	"github.com/fraunhoferfokus/deckschrubber/util"
```

Comment:
If you remove this dependency, you also need to remove the file, as it is not used anywhere else.